quipucords/scanner/acs/runner.py

```diff
@@ -0,0 +1,86 @@
+"""abc scan task runner for acs scanner."""
+
+from abc import ABCMeta
+from functools import cached_property
+
+from api.models import ScanJob, ScanTask
+from api.vault import decrypt_data_as_unicode
+from scanner.acs.api import ACSApi
+from scanner.runner import ScanTaskRunner
+
+
+class AcsTaskRunner(ScanTaskRunner, metaclass=ABCMeta):
```

Comment:
small inconsistency: in some places you are naming acs related classes ACSSomething (like ACSApi). Either change this one to `ACSTaskRunner` or change the other ones.

Reply:
I see you gave a thumbs up here, but I still see inconsistency in these names (ACSApi, for instance, is still around).

Reply:
ah, nevermind. the fix is in a fixup.